src/inmanta/moduletool.py

```diff
@@ -1614,7 +1591,7 @@ def __init__(self, module_path: str) -> None:
 
     def build(self, output_directory: str, dev_build: bool = False, byte_code: bool = False) -> str:
         """
-        Build the module and return the path to the build artifact.
+        Build the module using the pip system config and return the path to the build artifact.
```

Comment:
I can't find where the system's pip config is being used during the build, is this already implemented ?

Reply:
Ohh I see :+1: